File: learningCardApi/learningCardApi/Models/LearningCard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace learningCardApi.Models
{
    public class LearningCard
    {
        public int Id { get; set; }
        public string Front { get; set; }
        public string Back { get; set; }
        public int CardSetId { get; set; }

        public LearningCard(string front, string back, int cardSetId)
        {
            Front = front;
            Back = back;
            CardSetId = cardSetId;
        }
    }
}

File: learningCardApp/src/app/overzicht-manager/model/learning-card.ts
export class LearningCard {
    constructor 
    (
        private _front: string,
        private _back: string,
        private _cardSetId: number
    ) {}

    private _id: string;

    static fromJSON(json: any): LearningCard {
        const learnincard = new LearningCard(json.front, json.back, json.cardSetId);
        learnincard._id = json.id;
        return learnincard;
    }

    get front(): string {
        return this._front;
    }
    get back(): string {
        return this._back;
    }
    get cardSetId() {
        return this._cardSetId;
    }
    
}
